src/app/highscores/components/high-score-sidebar/high-score-sidebar.component.scss

```diff
@@ -8,15 +8,44 @@
   flex-direction: column;
   justify-content: space-around;
   align-items: center;
-  height: 250px;
+  height: 400px;
   width: 50px;
-  background: #c62828;
+  background: orange;
 }
 
 .content {
-  height: 250px;
+  height: 400px;
   width: 500px;
-  background: #1b5e20;
+  background: gold;
```

Comment:
You can use `mat-color($custom-theme-accent)` to get the actual accent color defined in our theme.

There are also options to select different hue levels of that color: https://material.angular.io/guide/theming-your-components#note-using-the-mat-color-function-to-extract-colors-from-a-palette